src/Microsoft.DotNet.Interactive.Formatting.Tests/Utility/StringExtensions.cs

```diff
@@ -25,7 +26,19 @@ public static string Crunch(this string s)
 
     public static string IndentHtml(this string html)
     {
-        return XElement.Parse(html).ToString();
+        try
+        {
+            return XElement.Parse(html).ToString();
+        }
+        catch (XmlException ex) when (ex.Message.Contains("multiple root elements"))
```

Comment:
Would we ever want to run tests on localized / non-ENU setups?

Reply:
It's unlikely but if so, it will produce a false negative rather than a false positive, which is good, and it will likely be pretty clear what happened. I did look for a better programmatic way to determine the specific cause of the `XmlException` but I didn't find anything.